micro-rdk/src/esp32/board.rs

```diff
@@ -84,104 +88,98 @@ impl EspBoard {
                     let analogs: Result<Vec<AnalogReaderType<u16>>, BoardError> = analogs
                         .iter()
                         .map(|v| {
-                            let adc1 = Arc::new(Mutex::new(AdcDriver::new(
-                                unsafe { ADC1::new() },
-                                &Config::new().calibration(true),
-                            )?));
+                            let adc1 = AdcDriver::new(unsafe { ADC1::new() })?;
+                            let config = AdcChannelConfig {
+                                attenuation: DB_11,
+                                calibration: Calibration::Line,
```

Comment:
What does that do and why do we set it to line?

Reply:
previously we did `&Config::new().calibration(true)`. The `Calibration` enum has three variants: `None`, `Line`, `Curve` according to the [source code](https://docs.esp-rs.org/esp-idf-hal/src/esp_idf_hal/adc.rs.html#532-553). However, only `None` and `Line` are available to our build. So I used `Line` to in place of the previous `calibration(true)`.

Reply:
make sense :ok_hand: